great-idea/css/index.css

```diff
@@ -63,4 +63,160 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+.container {
+  /* border: 1px solid red; */
+  height: 1536px;
+  width: 874px;
+  margin: 0 auto;
+}
```

Comment:
Container doesnt need height. 
Love how you set the width to constrain the rest of the elements inside!